src/parsers/manifest/dash/js-parser/node_parsers/__tests__/utils.test.ts

```diff
@@ -79,6 +80,35 @@ describe("dash parser helpers", function() {
     });
   });
 
+  describe("parseSupplementalCodec", () => {
+    it("should return the codec unchanged if there is only one codec", () => {
+      expect(parseSupplementalCodec("avc1.4d400d")).toEqual(["avc1.4d400d", null]);
+    });
+    it("should trim starting and ending whitespace",  () => {
+      expect(
+        parseSupplementalCodec("  avc1.4d400d  "))
+        .toEqual(["avc1.4d400d", null]);
+    });
+    it("should return comma-separated list if input is whitespace-separated" , () => {
+      expect(
+        parseSupplementalCodec("avc1.4d400d avc1.4d4015"))
+        .toEqual(["avc1.4d400d, avc1.4d4015", null]);
+    });
+    it("should return comma-separated value if input is alreadu comma-separated" , () => {
```

Comment:
=>     it("should return comma-separated value if input is already comma-separated" , () => {